### PR TITLE
feat: dynamically generate job offers based on lever API

### PR DESCRIPTION
**Pull request by miedziak**
Add generate job offers during gatsby build time mostly fo SEO purpose.
In addition, to keep it up to date for user, there is additional
request to lever API to fetch current data on client site.

Resolves: OCM-45

### CODE REVIEW

**Comment by mrinalwadhwa** [comment]
This is merged https://github.com/ockam-network/website/commit/da5f692d4f58aaa6918bbf85e661f39d7f8955b1

File: package.json [hunk 1]
```diff
@@ -30,6 +30,7 @@
     "@playlyfe/gql": "2.6.2",
     "@svgr/webpack": "5.1.0",
     "algoliasearch": "3.35.1",
+    "cross-fetch": "^3.0.4",
```
Comment (LWronka) [inline package.json]: should be removed `^`

File: package.json [hunk 2]
```diff
@@ -52,6 +53,7 @@
     "gatsby-remark-copy-linked-files": "2.1.36",
     "gatsby-remark-images": "3.1.42",
     "gatsby-source-filesystem": "2.1.46",
+    "gatsby-source-lever": "^2.1.24",
```
Comment (LWronka) [inline package.json]: as above

File: src/api/apiFetch.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+import fetch from 'cross-fetch';
+
+import config  from '../../config';
+
+const { api: { leverUrl } } = config;
+
+const toJson = resp => {
+  const json = resp.json();
+  if (resp.status >= 200 && resp.status < 300) {
```
Comment (LWronka) [inline src/api/apiFetch.js]: `fetch` deliver to us flag `ok` so you can made sth like this: 
if(response.ok) return resp.json()
throw new Error(respone.statusText)`

**Pull request closed without merging**